build/.goreleaser.yml

```diff
@@ -13,7 +16,25 @@ builds:
       - amd64
       - arm
       - arm64
-
+  - id: nri-nix-fips
+    main: ./src
+    binary: nri-docker
+    ldflags:
+      - -s -w -X main.integrationVersion={{.Version}} -X main.gitCommit={{.Commit}} -X main.buildDate={{.Date}}
+    env:
+      - CGO_ENABLED=1
+      - GOEXPERIMENT=boringcrypto
+      - >-
+        {{- if eq .Arch "arm64" -}}
+        CC=aarch64-linux-gnu-gcc
+        {{- end }}
+    goos:
+      - linux
+    goarch:
+      - amd64
+      - arm64
+    tags:
+      - fips
```

Comment:
did you run `go tool nm nri-docker | grep FIPS` to ensure they are FIPS ?(They should be but just as an extra check)

Reply:
Adding the output here for ref.
```log
root@1b2ee7eb3e5d:/app/dist# go tool nm nri-nix-fips_linux_amd64_v1/nri-docker | grep FIPS
  462b80 t BORINGSSL_FIPS_abort
  45ea80 t FIPS_mode_set
  45ea90 t FIPS_read_counter
  4015e0 T _cgo_14d622f8b473_Cfunc__goboringcrypto_FIPS_mode
  45ea70 t _goboringcrypto_FIPS_mode
  743660 T crypto/internal/boring._Cfunc__goboringcrypto_FIPS_mode.abi0
  e23c58 D crypto/internal/boring._cgo_14d622f8b473_Cfunc__goboringcrypto_FIPS_mode
  740800 T crypto/internal/boring/sig.FIPSOnly.abi0
  e26bc0 D crypto/tls.defaultCipherSuitesFIPS
  e26b80 D crypto/tls.defaultCurvePreferencesFIPS
  e26ba0 D crypto/tls.defaultSupportedSignatureAlgorithmsFIPS
  e26b60 D crypto/tls.defaultSupportedVersionsFIPS
root@1b2ee7eb3e5d:/app/dist# go tool nm nri-nix_linux_amd64_v1/nri-docker | grep FIPS
reading nri-nix_linux_amd64_v1/nri-docker: no symbol section
reading nri-nix_linux_amd64_v1/nri-docker: no symbols
root@1b2ee7eb3e5d:/app/dist# 
```

Reply:
Correct, thanks